### ISSUE TITLE
Cannot parse `time(n)` type in postgres

### ISSUE THREAD

**Issue by shusson** [taozhi8833998/node-sql-parser#364]
**Describe the bug**
Cannot parse time(n) type in postgres

**Database Engine**
postgres 12.1

**To Reproduce**

```sql
CREATE TABLE "foo"
(
    "Id" varchar(50) not null,
    "Test" time(7) null
);
```

```
peg$SyntaxError: Expected ")", ",", "--", "/*", "AUTO_INCREMENT", "COLLATE", "COLUMN_FORMAT", "COMMENT", "DEFAULT", "KEY", "NOT NULL", "NULL", "PRIMARY", "REFERENCES", "STORAGE", "UNIQUE", or [ \t\n\r] but "(" found.
```

- node-sql-parser 2.2.2
- node 12.14

**Expected behavior**
Can parse time(n) types.

**Comment by taozhi8833998** [comment]
ok, got it, I will support it asap.

**Comment by taozhi8833998** [comment]
try `2.2.3` version

**Comment by shusson** [comment]
👍 fixed in 2.2.3
